project/src/common.js

```diff
@@ -19,4 +21,43 @@ const getDate = function (comment) {
   };
 };
 
-export { getStarRating, getDate };
+const sortByDefault = function (someOffers) {
+  return someOffers;
+};
+
+const sortByPriceToHigh = function (someOffers) {
+  const sortedOffers = [...someOffers].sort((someOfferA, someOfferB) => someOfferA.price - someOfferB.price);
+  return sortedOffers;
+};
+
+const sortByPriceToLow = function (someOffers) {
+  const sortedOffers = [...someOffers].sort((someOfferA, someOfferB) => someOfferB.price - someOfferA.price);
+  return sortedOffers;
+};
+
+const sortByRatingFromTop = function (someOffers) {
+  const sortedOffers = [...someOffers].sort((someOfferA, someOfferB) => someOfferB.rating - someOfferA.rating);
+  return sortedOffers;
+};
```

Comment:
Код дублируется везде, это плохо

```
const getSortedItems = (items, compareFn) => {
  return items.slice().sort(compareFn);
};
```

```suggestion
const sortByPriceToHigh = function (someOffers) {
  return getSortedItems(someOffers, (someOfferA, someOfferB) => someOfferA.price - someOfferB.price)
};
```

Reply:
Не исправлено